development/kops/run_all.sh

```diff
@@ -13,18 +13,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -exo pipefail
```

Comment:
Ideally we wouldn't have to remove `x`

Reply:
The -x option risks deplaying credentials in a public log. We might want to leave the rest though.

Reply:
But with the latest revision we don't have that risk right?